### DISCUSSION TITLE
v1.12.13

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#2081] [分享 / Share]
📌**重要说明: 关于番剧区 3.0 新版播放器** (应该也包括电影和课程区)

> 对应 issue: #1992

仅做了一些小适配, 很多地方我也还没摸清楚:
- 网页全屏时可以隐藏顶栏和侧栏了
- 附加功能里的: 仅仅是凑合能跑, 按照当前视频提供功能, 还不能响应切换视频的变化
- 样式方面, 仅夜间模式做了一点, 没这么亮瞎眼了, 不过主题色啥的还没弄完. 其他样式类功能还全部未适配.

✨**新增**
- `简化直播间`支持屏蔽标题栏活动, 只留下最基本的信息. (#2019)

<div>

<img src="https://user-images.githubusercontent.com/26504152/126054673-b76b23a4-b2a6-4978-800b-abe4e5124b16.png">
<img src="https://user-images.githubusercontent.com/26504152/126054663-62cb7d83-bad0-42f4-bee2-e30d32037e90.png">

</div>

🐛**改进/修复**

- 修复`直播全屏包裹`和旁边的按钮没对齐.
- 修复`高分辨率图片`导致动态头像框在 Firefox 中显示异常. (#2030)
- `播放器触摸支持`适配播放器布局变动. (#2042)
- 夜间模式更新. (#1692, #2020, #2031)
- 快捷键中进度调整失效的问题: 尝试对查询 `video` 的地方附加了对 `bwp-video` 的查询. (#1931)
> 根据 @binsee 分享的播放器源码 (https://github.com/the1812/Bilibili-Evolved/issues/1931#issuecomment-878367037), `bwp-video` 应该是 Wasm 播放器的功能灰度, 虽然我还是没有被灰度到, 但先试试看 `bwp-video` 是否会把调用转发给内部的 `video` 吧

- 修复`瓜子换硬币`失效. (#1988)
- 修复`专栏图片导出`的命名出现`-1`. (#1973)
- 修复直播页面中`快速收起评论区`等动态功能失效. (#1900)
- 进一步完善专栏的夜间模式. (#1991)
- 修复`收起未关注人消息`打开时, `自定义顶栏`的消息数目仍然包含未关注人. (#1985)
- 分区数据更新, 影响包括`自定义顶栏`和`简化首页`. (#1511, #1918)
- `自定义顶栏`增加`番剧`入口. (#1938)

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v1.12.13'>v1.12.13</a>.</em>

**Comment by ZiSaAcZ7** [comment]
woc瑶山百灵？？？我是瑶姐25级粉丝牌！

**Reply by the1812** [reply]
🍊!

**Reply by RingingVolcano** [reply]
🍊!

**Reply by ZiSaAcZ7** [reply]
> 🍊!

橘子！

**Reply by RingingVolcano** [reply]
> > 🍊!
> 
> 橘子！
> 

哪里都有越共（
